packages/frontend/src/pages/AttesterPage.tsx

```diff
@@ -0,0 +1,137 @@
+import { useContext, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { observer } from 'mobx-react-lite'
+import State from '../contexts/state'
+import Tooltip from '../components/info/Tooltip'
+import InfoCard from '../components/info/InfoCard'
+import AttestationCard from '../components/events/attesterEvents/AttestationCard'
+import UserCard from '../components/events/attesterEvents/UserCard'
+import Footer from '../components/layout/footer'
+
+
+export default observer(() => {
+  const { info } = useContext(State)
+  const { id } = useParams()
+  const [Selected, setSelected] = useState(0)
+  const toggleSelected = (Id: number) => setSelected(Id)
+
+  return (
+    <>
+      <div className='container'>
+
+        <div className='left-container'>
+          <h3>About</h3>
+          <div className='image-placeholder'></div>
+          <h1>project name</h1>
+          <a style={{color: '#669294'}}href="https://github.com/Unirep/create-unirep-app" target='blank'>https://url-of-attester</a>
+          <p style={{fontSize: '1.1em'}}>Lorem ipsum dolor sit amet consectetur. Molestie elit sit ut magna vel quis ultricies auctor porttitor. Mauris scelerisque tellus eleifend nunc id purus in. Tempus fames lacus feugiat lectus nibh morbi sapien et.</p>
+          <h3>Attester {id}</h3>
+          <div>Connected to a server with the following info:</div>
+          <ul>
+            <li>Unirep Address: {info.UNIREP_ADDRESS}</li>
+            <li>Provider URL: {info.ETH_PROVIDER_URL}</li>
+          </ul>
+          <div className='info-card'>
+            <div className='flex'>
+              <h4>Attester Information</h4>
+              <Tooltip text='Some info goes here' maxWidth={200} />
+            </div>
+            <div className='flex'>
+              <h5>ID</h5>
+              <h6>2</h6>
+            </div>
+            <div className='flex'>
+              <h5>Deployed at</h5>
+              <h6>Jan/20/2023</h6>
+            </div>
+            <div className='flex'>
+              <h5>Address</h5>
+              <h6>0xdC9...Bc9</h6>
+            </div>
+          </div>
+        </div>
+
+        <div className='right-container'>
+          <h3>Overview</h3>
+          <div className='info-grid'>
+            <InfoCard heading='Epochs Processed' value={'61'} valueIsNum={true}/>
+            <InfoCard heading='Total Rep Given' value={'2,109'} valueIsNum={true}/>
+            <InfoCard heading='Total Users Signed Up' value={'20'} valueIsNum={true}/>          
+            <InfoCard heading='Hashchain Status' value={'Processing'} valueIsNum={false}/>
+          </div>
+          {Selected === 0 ? (
```

Comment:
This ternary is quite large, getting into separate component territory.

Reply:
👍 agreed